src/runtime/Makefile

```diff
@@ -167,6 +167,10 @@ DEFDISABLEGUESTEMPTYDIR := false
 DEFAULTEXPFEATURES := []
 
 DEFDISABLESELINUX := false
+#Default SeccomSandbox param
+#The same default policy is used by libvirt
+#More explanation on https://lists.gnu.org/archive/html/qemu-devel/2017-02/msg03348.html
```

Comment:
elevateprivileges=deny does not work with demonize which we use.

Reply:
@mcastelino Thanks for pointing this out. I just tested and confirmed `elevateprivileges=deny` does not work with demonize